frontend/vite.config.ts

```diff
@@ -24,5 +24,12 @@ export default defineConfig(() => {
       port: 8000,
       open: true,
     },
+    define: {
+      __BUILD_TIMESTAMP__: new Date()
+        .toISOString()
+        .split(".")[0]
+        .split("T")
+        .join(" "),
+    },
```

Comment:
könntest du an dieser stelle nicht auch ein `git describe --tags` ausführen und ein fallback auf den Timestamp haben?

Reply:
Problem with this is that we're building in docker, and git hasn't been working in there

Reply:
In the backend using an extra step to define the version externally: https://github.com/ingef/conquery/blob/develop/Dockerfile#L2

Reply:
Nice, will have to adopt that soon for frontend as well